src/main/java/com/nike/cerberus/client/CerberusClient.java

```diff
@@ -65,6 +76,13 @@
     private final Gson gson = new GsonBuilder()
             .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
             .disableHtmlEscaping()
+            .registerTypeAdapter(DateTime.class, new JsonDeserializer<DateTime>() {
```

Comment:
Just a question. Is this needed?

Reply:
This may not be necessary in later versions of Gson and also Java (e.g. Java 8). However, we have pinned the client to Java 7.

The deserializer is required to tell Gson how to deserialize the timestamps returned with file metadata.